modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/Resolver.scala

```diff
@@ -111,7 +111,8 @@ final case class Resolver[F[_]](repos: List[Registry], cache: Option[ResolverCac
   def listSchemasResult(
     vendor: Vendor,
     name: Name,
-    model: Model
+    model: Model,
+    mustIncludeKey: Option[SchemaKey] = None
```

Comment:
As this is public method, such change would mean breaking compatibility? So to avoid that, we could either: 

* don't touch this method, then basically copy all that content but with `mustIncludeKey` to the new `listSchemasLike` method. 
* extract this new method body to private method, `listSchemasResult` would call it with `None`, `listSchemasLike` with `Some`

Reply:
I have a suspicion that this method should not be public. Cached/NotCached should be transparent for the consumers.